browser-interface/packages/shared/session/index.ts

```diff
@@ -23,6 +24,8 @@ export const storeSession: (session: StoredSession) => Promise<void> = async (se
 export const deleteSession = async (userId?: string) => {
   if (userId) {
     await removeFromPersistentStorage(sessionKey(userId))
+
+    await SingleSignOn.clearIdentity(userId)
```

Comment:
Afaik the userId is the user address.
When logging out this will clear the identity from the iframe